pam/internal/adapter/model.go

```diff
@@ -201,7 +228,7 @@ func (m *UIModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		log.Info(context.TODO(), "UILayoutReceived")
 
 		return m, tea.Sequence(
-			m.authenticationModel.Compose(m.currentSession.brokerID, m.currentSession.sessionID, msg.layout),
+			m.authenticationModel.Compose(m.currentSession.brokerID, m.currentSession.sessionID, m.currentSession.encryptionKey, msg.layout),
```

Comment:
> 120 chars wide :)

Reply:
Not our limit :p (as already argued in other PRs), here we have 2 sequences and it reads better that way

Reply:
So we said, didn't we? :)

Reply:
I don't think we never acknowledged to it, or if so, we would have a checker. We have lines of 130 characters in other projects and it seems that didn’t create any issue inside desktop enterprise/wsl team.

So I suggest either we all discuss this to have it as a rule in our standup, and if so, write a linter, ensure that all projects pass to it, or just deal with it :) (for now, I would argue that the line is readalable and wrapping make it harder to parse the sequence itself)